config/304-xslttransform.yaml

```diff
@@ -0,0 +1,136 @@
+# Copyright 2021 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: xslttransforms.flow.triggermesh.io
+  labels:
+    triggermesh.io/crd-install: 'true'
+spec:
+  group: flow.triggermesh.io
+  scope: Namespaced
+  names:
+    kind: XsltTransform
+    plural: xslttransforms
+    categories:
```

Comment:
If we're going to start using lowercase for acronyms in Kubernetes kinds, we should also rename all other types that use acronyms (AWS-> Aws, etc.)